docs/getting-started/development.md

```diff
@@ -18,31 +22,32 @@ $ vagrant plugin install vagrant-group
 # Automatically manage Guest Additions versions
 $ vagrant plugin install vagrant-vbguest
 ```
-
-You'll need to structure your Moodle development environment appropriately. Ubiquitous sets up a synced folder for `../Moodle` relative to itself to get your source code to the application server:
-
-```
-.
-├── Moodle
-└── Ubiquitous
-```
-
 Start up all of the machines necessary for a testing environment:
 
 ```
+$ cd moodle-ubiquitous
 $ vagrant group up dev
 ```
 
-The first time you start the servers, and whenever you make changes to the Salt states, you'll need to apply the states to the machines:
+The [Vagrantfile](/Vagrantfile) makes assumptions about the host environment, including:
+ * Moodle itself is installed in `../Moodle` relative to Ubiquitous
+ * standard ports are available (for e.g. SSH forwarding)
+
+Use Vagrantfile.config and / or Vagrantfile.local to override as needed.
+
+The first time you start the servers, and on changes to a Salt state definition, converge the machine state:
 
 ```
-# Provision the Salt master first, opening the ports necessary for
-# master-minion configuration
+# Converge primary VM (Salt master) first
+# This opens necessary ports for master-minion configuration
 $ vagrant ssh --command 'sudo salt salt state.apply'
 
 # Then converge the rest of the machines
 $ vagrant ssh --command 'sudo salt '*' state.apply'
 ```
+The above may take some time to complete; and can time out "Minion did not return. [No response]" while the minion is still configuring.
+Re-try, or alternatively, perform manually for a specific minion:
+$ vagrant ssh app-debug-1 --command 'sudo salt-call state.apply'
```

Comment:
This isn't a timeout -- it's caused by the `iptables` configuration. I _think_ the issue is the order in which the rules are applied, where we deny all incoming traffic _before_ the rule to allow established connections.

Please clarify the comment to reflect this and fence the code block with three backticks, as with other commands in this document.